### PR TITLE
Update us.m3u

### LINKED ISSUE #7129: Add: PHX TV - (Phoenix TV)
### Channel Name

PHXTV

### Channel ID

_No response_

### Website

https://www.phoenix.gov/phxtv

### Stream URL

https://granicusliveus8-a.akamaihd.net/phoenix/G1498_002/playlist.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #7546: Add: EMT Media TV - Los Angeles, CA
### Channel Name

EMT Media TV

### Channel ID

_No response_

### Website

https://emtmediatv.com/watch-live-tv/

### Stream URL

_No response_

### Notes

Please adding the m3u8 link

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #7664: Add: Reelz Channel | (US)
### Channel Name

Reelz Channel

### Channel ID

ReelzChannel.us

### Website

reloltv.com

### Stream URL

https://bcovlive-a.akamaihd.net/c733a9aa448a4a44a10c527c6f5bf7a4/us-east-1/5245389775001/playlist.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #7773: Add: Pan Armenian TV
### Channel Name

Pan Armenian TV

### Channel ID

panarmenian.tv

### Website

https://www.panarmenian.tv/watch-live/

### Stream URL

_No response_

### Notes

Country: United States

Language: Armenian

Category: General

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #7789: Univision streams -> to be added.
Univision East https://unidfp-nlds155.global.ssl.fastly.net/nlds/univisionnow/univision_east2/as/live/univision_east2_hd_pc.m3u8?t1=
Univision West https://unidfp-nlds138.global.ssl.fastly.net/nlds/univisionnow/univision_dal2/as/live/univision_dal2_hd_pc.m3u8?t1=
UniMás East https://unidfp-nlds164.global.ssl.fastly.net/nlds/univisionnow/unimas_east2/as/live/unimas_east2_hd_pc.m3u8?t1=
TUDN https://unidfp-nlds141.global.ssl.fastly.net/nlds/univisionnow/udn2/as/live/udn2_hd_pc.m3u8?t1=
Galavisión East https://unidfp-nlds159.global.ssl.fastly.net/nlds/univisionnow/galavision_east/as/live/galavision_east_hd_pc.m3u8?t1=
Galavision West https://unidfp-nlds159.global.ssl.fastly.net/nlds/univisionnow/galavision_west/as/live/galavision_west_hd_pc.m3u8?t1=
Fútbol https://unidfp-nlds153.global.ssl.fastly.net/nlds/univisionnow/futbol/as/live/futbol_hd_pc.m3u8?t1=
Despierta https://unidfp-nlds140.global.ssl.fastly.net/nlds/univisionnow/challenge28/as/live/challenge28_hd_pc.m3u8?t1=
Novelas https://unidfp-nlds166.global.ssl.fastly.net/nlds/univisionnow/novelas2plist/as/live/novelas2plist_hd_pc.m3u8?t1=
Delicioso https://unidfp-nlds139.global.ssl.fastly.net/nlds/univisionnow/cocina2/as/live/cocina2_hd_pc.m3u8?t1=

_Originally posted by @robocopz in https://github.com/iptv-org/iptv/issues/7781#issuecomment-1188776506_

### LINKED ISSUE #7827: Add: Univision HD | (Colombia)
### Channel Name

Univision

### Channel ID

_No response_

### Website

_No response_

### Stream URL

http://190.61.41.182:8000/play/a37m/index.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #7876: Add: Grit Xtra
### Channel Name

Grit Xtra

### Channel ID

GritXtra.us

### Website

https://www.grittv.com/gritxtra-schedule/

### Stream URL

https://scripps-gritxtra-1-us.vizio.wurl.tv/playlist.m3u8

### Notes

This is not the Grit channel, it's a second Grit channel which features a different line up. It's also available on TCL TV, I just don't know the streaming URL

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #7881: Add: Earthx TV
### Channel Name

EarthxTV

### Channel ID

_No response_

### Website

https://earthx.org/earthxtv-schedule/

### Stream URL

https://ov.ottera.tv/live/master2300000.m3u8;session=earth_et

### Notes

There is an EPG on the website but I'm not sure how to extract it. I think it's American, but not sure either.

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #7989: Add: PBS Arkansas
### Channel Name

Arkansas PBS

### Channel ID

KETSDT1.us

### Website

https://www.myarkansaspbs.org/watch

### Stream URL

https://ketsdt.lls.pbs.org/out/v1/03c094dbd7874a4a8c3fe9fb10081bdb/index_3.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #7992: Add: Assemblee Nationale du Quebec
### Channel Name

Assemblee Nationale du Quebec

### Channel ID

AssembleenationaleduQuebec.ca

### Website

http://m.assnat.qc.ca/en/index.html

### Stream URL

https://cdn3.wowza.com/5/UHh3aHM1bTBsY1Uw/diffusion/canal17/playlist.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #7993: Add: WSTO
### Channel Name

WSTO

### Channel ID

_No response_

### Website

https://wsto.tv/

### Stream URL

https://5d80890572342.streamlock.net:1936/live/wstoLive/chunklist_w459911506.m3u8

### Notes

This is a valid American PEG station from Stoughton, WI; as such, it is not listed in your formal list.
I hope you accept it.

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #8007: Add: KTOO 360TV 3 (FNX)
### Channel Name

KTOO DT3

### Channel ID

KTOODT3.us

### Website

https://www.ktoo.org/360TV/

### Stream URL

https://cdn3.wowza.com/5/ODB6NmF5K3l4T1h5/persis/2147483647_360tv_247/chunklist.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #8020: Add: City TV - Capital City Connection
### Channel Name

City TV - Capital City Connection

### Channel ID

CapitalCityConnectionMontgomery.us

### Website

https://www.montgomeryal.gov/home

### Stream URL

https://vblive-c.viebit.com/5f0d9ca5-4e85-4c01-a426-9ec8d44c2c9c/chunklist.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #8031: Add: Fayetteville Government Channel
### Channel Name

Fayetteville Government Channel

### Channel ID

_No response_

### Website

https://www.fayetteville-ar.gov/510/Media-Services-PEG-TV

### Stream URL

https://youtu.be/yg2B7Nzryek

### Notes

From Fayetteville, AR when available; PEG channel

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #8047: Add: TV4 Bullhead City, AZ
### Channel Name

TV4 Bullhead City, AZ

### Channel ID

_No response_

### Website

https://www.bullheadcity.com/government/tv4

### Stream URL

https://cdn3.wowza.com/5/UWpORHhLSEs5SkJs/bullheadcity/G0860_001/chunklist.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #8049: Add: LHCAZTV
### Channel Name

LHCAZTV

### Channel ID

_No response_

### Website

https://www.lhcaz.gov/channel-4

### Stream URL

https://cdn3.wowza.com/5/cHYzekYzM2kvTVFH/lakehavasucity/G0643_003/chunklist.m3u8

### Notes

From Lake Havasu, AZ

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #8069: Add: Prescott Valley TV
### Channel Name

Prescott Valley TV

### Channel ID

_No response_

### Website

https://www.prescottvalley-az.gov/423/PV-TV---Channel-1056

### Stream URL

https://live-h1.frontlayer.net/fl207891/live/chunks.m3u8

### Notes

From Prescott Valley, AZ

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #8070: Add: Channel 11
### Channel Name

Channel 11

### Channel ID

_No response_

### Website

https://www.scottsdaleaz.gov/scottsdale-video-network/live-stream

### Stream URL

https://cdn3.wowza.com/5/RXJNMFI3VlVkOEFP/scottsdale/G0040_007/chunklist.m3u8

### Notes

From Scottsdale, AZ

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #8071: Add: ACTV
### Channel Name

ACTV

### Channel ID

_No response_

### Website

https://www.azleg.gov/actvlive/

### Stream URL

https://cdn3.wowza.com/5/ODB6NmF5K3l4T1h5/persis/6361162879_AZ-247.stream_247/chunklist.m3u8

### Notes

From Phoenix, AZ

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #8085: Add: Tempe Channel 11
### Channel Name

Tempe Channel 11

### Channel ID

Tempe11.us

### Website

https://www.tempe.gov/government/communication-and-media-relations/tempe-11

### Stream URL

https://cdn3.wowza.com/5/cFh0V0QwUVc4SDl2/tempe/G0355_003/chunklist.m3u8

### Notes

From Tempe, AZ

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #8086: Add: AWC TV
### Channel Name

AWC TV

### Channel ID

_No response_

### Website

https://www.azwestern.edu/communications-and-marketing/television-services

### Stream URL

https://arizonawesterncol.secure.footprint.net/egress/bhandler/arizonawesterncol/streama/chunklist_b2368512.m3u8

### Notes

From Yuma, AZ

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #8087: Add: edTV
### Channel Name

edTV

### Channel ID

_No response_

### Website

https://www.azwestern.edu/communications-and-marketing/television-services

### Stream URL

https://arizonawesterncol.secure.footprint.net/egress/bhandler/arizonawesterncol/streamb/chunklist_b2368512.m3u8

### Notes

From Yuma, AZ

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #8094: Add: Knoxville Community Media
### Channel Name

Knoxville Community Media

### Channel ID

_No response_

### Website

https://www.ctvknox.org/

### Stream URL

https://knoxville-tn.secure.footprint.net:443/egress/bhandler/knoxvilletn/streama/chunks.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #8097: Add: EPlusTV 6 [Jackson, Tennessee] | (US)
### Channel Name

EPlus TV

### Channel ID

_No response_

### Website

https://www.eplustv6.com/

### Stream URL

https://jacksonenergy.secure.footprint.net/egress/bhandler/jacksonenergy/streama/index.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### PR DESCRIPTION
**Pull request by freearhey**
Added new links from issues.

Closes #8031, closes #8094, closes #8087, closes #8086, closes #8085, closes #8049, closes #8047, closes #8020, closes #8007, closes #7993, closes #7876, closes #7827, closes #7789, closes #7664, closes #7546, closes #7129, closes #8097, closes #8071, closes #8070, closes #8069, closes #7992, closes #7989, closes #7773, closes #7881

### CODE REVIEW

**Review by what-a — DISMISSED** [review]

**Comment by Central302** [comment]
[To-Do]maybe, Univision Dal2 Meaning  Univision DFW?

File: streams/us.m3u [hunk 1]
```diff
@@ -2077,3 +2077,69 @@ https://www.livedoomovie.com/02_AXNHD_720p/chunklist.m3u8
...
+#EXTINF:-1 tvg-id="",Despierta
+https://unidfp-nlds140.global.ssl.fastly.net/nlds/univisionnow/challenge28/as/live/challenge28_hd_pc.m3u8?t1=
+#EXTINF:-1 tvg-id="",Novelas
+https://unidfp-nlds166.global.ssl.fastly.net/nlds/univisionnow/novelas2plist/as/live/novelas2plist_hd_pc.m3u8?t1=
+#EXTINF:-1 tvg-id="",Delicioso
+https://unidfp-nlds139.global.ssl.fastly.net/nlds/univisionnow/cocina2/as/live/cocina2_hd_pc.m3u8?t1=
+#EXTINF:-1 tvg-id="ReelzChannel.us",Reelz Channel
+https://bcovlive-a.akamaihd.net/c733a9aa448a4a44a10c527c6f5bf7a4/us-east-1/5245389775001/playlist.m3u8
+#EXTINF:-1 tvg-id="",EMT Media TV Los Angeles CA
+https://mediaplatform.streamingmediahosting.com/p/14135/sp/1413500/playManifest/entryId/0_lkq7fay8/format/applehttp/protocol/https/uiConfId/6710347/a.m3u8?referrer=aHR0cHM6Ly9lbXRtZWRpYXR2LmNvbQ==&playSessionId=85851bc0-fd37-a365-4004-da87b1d0035e
```
Comment (LaneSh4d0w) [inline streams/us.m3u:2125]:
```suggestion
https://mediaplatform.streamingmediahosting.com/p/14135/sp/1413500/playManifest/entryId/0_lkq7fay8/format/applehttp/protocol/https/uiConfId/6710347/a.m3u8
```
unnecessary referer and sessionid

Comment (LaneSh4d0w) [reply streams/us.m3u:2125]: alternative link being https://hwlive.streamingmediahosting.com/14135-live/0_lkq7fay8/index.m3u8

File: streams/us.m3u [hunk 2]
```diff
@@ -2077,3 +2077,69 @@ https://www.livedoomovie.com/02_AXNHD_720p/chunklist.m3u8
...
+https://cdn3.wowza.com/5/ODB6NmF5K3l4T1h5/persis/2147483647_360tv_247/playlist.m3u8
+#EXTINF:-1 tvg-id="",WSTO
+https://5d80890572342.streamlock.net:1936/live/wstoLive/playlist.m3u8
+#EXTINF:-1 tvg-id="GritXtra.us",Grit Xtra
+https://scripps-gritxtra-1-us.vizio.wurl.tv/playlist.m3u8
+#EXTINF:-1 tvg-id="",Univision
+http://190.61.41.182:8000/play/a37m/index.m3u8
+#EXTINF:-1 tvg-id="",Univision East
+https://unidfp-nlds155.global.ssl.fastly.net/nlds/univisionnow/univision_east2/as/live/univision_east2_hd_pc.m3u8?t1=
+#EXTINF:-1 tvg-id="",Univision West
```
Comment (Central302) [inline streams/us.m3u:2104]: This Channel is Univision DFW (EST)
Uni_DFW

Comment (Central302) [reply streams/us.m3u:2104]: univision_dfw2

Comment (Central302) [inline streams/us.m3u:2104]: This Stream is Univision DFW
Uni_DFW

Comment (freearhey) [reply streams/us.m3u]: updated

File: streams/us.m3u (hunk 1 above)
Comment (freearhey) [reply streams/us.m3u]: added

**Comment by freearhey** [comment]
> @freearhey (WGBO-DT1 Unisvion Chicago,US IP ONLY)
> https://unidfp-nlds154.global.ssl.fastly.net/nlds/univisionnow/univision_chi2/as/live/univision_chi2_hd_pc.m3u8?t1=

Added.

**Review by BellezaEmporium — APPROVED** [review]

**Review by Central302 — APPROVED** [review]

**Pull request merged**